src/main/webapp/javascript/user-page.js

```diff
@@ -30,10 +29,42 @@ async function loadUserRecipes() {
           elementsToAddToImageDiv.forEach(elem => imageDivs[i].appendChild(elem));
         }
       }
+      addDeleteFunctionality(recipes);
+      // TODO: add edit functionality
     }
   }
 }
 
+function addDeleteFunctionality(recipes){
```

Comment:
Need to add comment for function

Reply:
isn't it pretty obvious based on the function name and parameter?